modules/apps/29-fee/keeper/grpc_query.go

```diff
@@ -1,5 +1,61 @@
 package keeper
 
-// TODO
+import (
+	"context"
 
-//var _ types.QueryServer = Keeper{}
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// IncentivizedPackets implements the IncentivizedPackets gRPC method
+func (q Keeper) IncentivizedPackets(c context.Context, req *types.QueryIncentivizedPacketsRequest) (*types.QueryIncentivizedPacketsResponse, error) {
```

Comment:
nit: I know that we're using the var `q` for the transfer query functions also but I find this confusing. 

@colin-axner is there any reason for this? I feel like using `k` or `keeper` like the rest of the SDK modules is easier to understand.

Reply:
I'm not sure the reason. `k` sounds good